app/services/create_vehicle.rb

```diff
@@ -0,0 +1,8 @@
+class CreateVehicle
```

Comment:
I think a better name (if you like to use services, see my first comment) it could be `CreateStockedVehicle`, because the unique mean of this service is to create a service with `status` equals to `:stock` :P